android/src/main/java/io/mosip/tuvali/transfer/Chunker.kt

```diff
@@ -34,8 +34,7 @@ class Chunker(private val data: ByteArray, private val maxDataBytes: Int = DEFAU
   private fun chunk(seqNumber: Int): ByteArray {
     val fromIndex = seqNumber * effectivePayloadSize
 
-    return if (seqNumber == (totalChunkCount - 1).toInt() && lastChunkByteCount > 0) {
-      Log.d(logTag, "fetching last chunk")
+    return if (seqNumber == (totalChunkCount - 1).toInt() && lastChunkByteCount > 0) { Log.d(logTag, "fetching last chunk")
```

Comment:
_please see if this code-change is correctly formatted as per requirements_

Reply:
formatted it.